Makefile

```diff
@@ -34,13 +40,14 @@ download-ext:
 clean-ext:
 	$(MAKE) -C src_ext distclean
 
-clean: fastclean
```

Comment:
But for this change, I would have put this in #2935. The dependency on fastclean introduces a dependency on ocp-build... this isn't necessary the best way of handling that optional dependency! Does it *have* to be here and, if so, could ocp-build detection be put in configure instead?